snapshot_manager/main.py

```diff
@@ -56,7 +56,6 @@ def main():
     if cmd == "check":
         cfg.github_repo = args.github_repo
         cfg.datetime = args.datetime
-        cfg.strategy = args.strategy
```

Comment:
There is no `strategy` member in `config.Config` but only `build_strategy`. The strategy is already filled when selecting the strategy [here](https://github.com/fedora-llvm-team/llvm-snapshots/pull/1164/commits/116ccbadac5125e555c38cccdafdfed57bdb2804#diff-e735df349e8a75dd96a90e260766907bedb904e7828748af2981ef729e027288L48-L54).